packages/run-protocol/src/vpool-xyk-amm/addPool.js

```diff
@@ -47,15 +47,6 @@ export const makeAddIssuer = (
       harden({ decimalPlaces: 6 }),
     );
     await zcf.saveIssuer(secondaryIssuer, keyword);
-    const issuer = zcf.getIssuerForBrand(secondaryBrand);
-    console.log(
-      'Saved issuer',
-      secondaryIssuer,
-      'to keyword',
-      keyword,
-      'and got back',
-      issuer,
-    );
```

Comment:
Please put this in a separate PR.

Reply:
Why? Same reason as https://github.com/Agoric/agoric-sdk/pull/5538#discussion_r895083300 ?

Reply:
Basically the same, but it's a more substantive change, and in the AMM. I don't think this is a trivial enough unrelated change to leave in a PR for vaults.